pkg/api/unversioned/generated.proto

```diff
@@ -21,6 +21,7 @@ syntax = 'proto2';
 
 package k8s.io.kubernetes.pkg.api.unversioned;
 
+import "k8s.io/kubernetes/pkg/runtime/generated.proto";
```

Comment:
We're not filtering imports in the protobuf, so if we touch a type that
requires it it'll get added.

On Thu, May 5, 2016 at 2:29 AM, Wojciech Tyczynski <notifications@github.com
> wrote:

> In pkg/api/unversioned/generated.proto
> <https://github.com/kubernetes/kubernetes/pull/24710#discussion_r62152408>
> :
>
> > @@ -21,6 +21,7 @@ syntax = 'proto2';
> >
> >  package k8s.io.kubernetes.pkg.api.unversioned;
> >
> > +import "k8s.io/kubernetes/pkg/runtime/generated.proto";
>
> Where does those lines come from? You are not touching generator in this
> PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24710/files/e416be61b90b5dc461dc8a84a94d6ecd0d76c216#r62152408>
>